src/main/groovy/uk/ac/kcl/model/Document.groovy

```diff
@@ -11,13 +11,19 @@ import java.sql.Timestamp
 class Document {
 
     //generic fields
-    String databaseName
-    String databaseSchema
+    //String databaseName
+    //String databaseSchema
```

Comment:
where is this being used databaseSchema? Not sure if it is anywhere.

Reply:
I don't see it anywhere in the code, hence I commented it out in case anyone could have any particular idea in mind or whether it is used in any external module build on top of pipeline?
Shall I remove it?